core/src/main/scala/org/apache/spark/storage/StorageStatusListener.scala

```diff
@@ -28,15 +35,39 @@ import org.apache.spark.scheduler._
  *
  * This class is thread-safe (unlike JobProgressListener)
  */
+
+@DeveloperApi
+object StorageStatusListener {
+  val TIME_TO_EXPIRE_KILLED_EXECUTOR = "spark.ui.timeToExpireKilledExecutor"
```

Comment:
do we really need to have this class? how about just exposing this string to the end user?